### PR TITLE
cleanup test directory

### PR DESCRIPTION
**Pull request by IronPan**
/assign @Bobgy

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1914)

### CODE REVIEW

File: test/presubmit-tests-gce-minikube.sh [hunk 1]
```diff
@@ -1,87 +0,0 @@
-#!/bin/bash -ex
```
Comment (Ark-kun) [inline test/presubmit-tests-gce-minikube.sh]: This script is pretty generic - it just provisions a VM and runs arbitrary scripts on it. We might want to keep it (and just rename).

**Comment by Ark-kun** [comment]
This `test/presubmit-tests-gce-minikube.sh` script is pretty generic - it just provisions a VM and runs arbitrary scripts on it. We might want to keep it (and just rename).

File: test/presubmit-tests-gce-minikube.sh (hunk 1 above)
Comment (IronPan) [reply test/presubmit-tests-gce-minikube.sh]: I'm cleanup the files that are not being used. If this file is needed we could bring it back.

**Comment by Bobgy** [comment]
@IronPan deploy-kubeflow and deploy pipeline are still used by post submit tests. Is this a mistake?

**Comment by Ark-kun** [comment]
deploy_kubeflow is still used by postsubmit tests

**Comment by gaoning777** [comment]
Alexey is correct. postsubmit is still using the deploy_kubeflow

**Comment by Bobgy** [comment]
Discussed with @IronPan, I will migrate postsubmit tests to lite deployment anyway. If I can finish that soon, we can just wait for that. Otherwise, let's fix postsubmit tests.

**Pull request merged**